frontend/src/pages/interviewschedule.js

```diff
@@ -98,21 +146,30 @@ class InterviewSchedule extends Component {
     }
   }
 
-  async componentDidMount() {
+  populateInterviewSchedules = async () => {
     const res = await getInterviewSchedule()
+    if (res.code !== 200) {
```

Comment:
don't use `code`, use the `success` field; I know that you only use 200 in your backend but it's better practice to generalize beyond a specific response code